### ISSUE TITLE
Add documentation to integrate with Python Jupyter notebooks

### ISSUE THREAD

**Issue by rjurney** [lappsgrid-incubator/jupyter-groovy-kernel#11]
I'm a Python and Groovy user who would like to do both in the same notebook. Does the system you wrote support this? If so, how? Can you direct me or direct everyone in the docs to somewhere we can learn the broader context of how to use this system without implementing our own DSL? I just want to run Groovy/Gremlin queries inside a notebook with Python code.

**Comment by ksuderman** [comment]
No, our kernel only runs Groovy code.  It has been some time since we have worked on the kernel, but what you want might be possible.  You would either need a Groovy kernel that is capable of running Python code or a Python kernel capable of running Groovy code and some way to distinguish which language is being used in a cell.

There is some discussion of multiple language kernels at https://github.com/jupyterlab/jupyterlab/issues/2815 and you may want to take a look at https://vatlab.github.io/sos-docs/ (linked in the above issue as well).

However, I am not sure what you mean by, "learn the broader context of how to use this system without implementing our own DSL?".  You should be able to use the Groovy kernel as-is without writing a DSL, and writing a DSL is mostly just providing a custom CompilerConfiguration and MetaClass objects.
